### PR TITLE
rpc: update swagger docs to openapi 3.0

### LINKED ISSUE #4092: https://tendermint.com/rpc/#/Info/blockchain "Try it out" functionality gives network error
https://tendermint.com/rpc/#/Info/blockchain has a "Try it out"-button. Clicking it and then pressing the blue "Execute"-button spins a "LOADING" thing for a while and then says "TypeError: NetworkError when attempting to fetch resource.".

(This may be the wrong place to report problem  but there is no obvious feedback or contact information at the https://tendermint.com/rpc/ page).

### LINKED ISSUE #4144: swagger: upgrade to OpenAPI 3 standard
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md
https://swagger.io/docs/specification/basic-structure/

### PR DESCRIPTION
**Pull request by marbar3778**
- closes #4144
- closes #4092 

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by marbar3778** [comment]
full support from dredd is not available currently: https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/STATUS.md, do we want to disable it for now untill it becomes stable?

File: rpc/swagger/swagger.yaml [hunk 1]
```diff
@@ -828,1296 +892,1066 @@ paths:
...
+          x-example: "0F2908883A105C793B74495EB7D6DF2EEA479ED7FC9349206A65CB0F9987A0B8"
+        app_hash:
+          type: string
+          x-example: "223BF64D4A01074DC523A80E76B9BBC786C791FB0A1893AC5B14866356FCFD6C"
+        last_results_hash:
+          type: string
+          x-example: ""
+        evidence_hash:
+          type: string
+          x-example: ""
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]: I thought there's no such thing as `x-example`?

**Comment by melekes** [comment]
> do we want to disable it for now untill it becomes stable?

sure. just don't forget to create an issue to enable it later

**Comment by marbar3778** [comment]
https://github.com/tendermint/tendermint/issues/4229

**Pull request merged**